www/cli.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
Right.  I'll move it to /cli and probably rename it to joomla.php.

I don't like this www folder.  I think the web entry point should by default be in the web root (renamed to api.php), with the option that the site builder can move it anywhere they want (eg. into a folder mapped to a subdomain).